tests/acceptance/custom_browser/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+
+from selene import Config, Browser
+
+
+@pytest.fixture(scope='session')
+def session_driver():
+    chrome_driver = webdriver.Chrome(ChromeDriverManager().install())
+    yield chrome_driver
+    chrome_driver.quit()
+
+
+@pytest.fixture(scope='function')
+def session_browser(session_driver):
```

Comment:
since this is acceptance kind of test... we should more close to the user style in real life...
in real case, most probably it will be too much to name a browser as session_browser
too long name

let's name it here as just `browser`

Reply:
okay :) but it will be intersect with imports and juniors could fail a bit if they will try to use both in one module :)

Reply:
should not be a problem
you either pass browser as a fixture to a test function
and don't need an import
and don't need another "browser from selene imports"
or don't pass and use imported browser:)

so kind of it should be pretyy obvious from the context what to use... 

usually it's also one style per project that somebody is going to use...

Reply:
Agreed. Done it in 33992b29b0efaf88cba3bf37509d24e6b6401ad4